### PR TITLE
feat: add audit log to the centralized-logging module

### PR DESCRIPTION
**Pull request by LuizSDCit**
Fixes #

Add the audit log to the centralized-logging module.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: modules/centralized-logging/variables.tf [hunk 1]
```diff
@@ -69,3 +74,9 @@ variable "kms_key_protection_level" {
   type        = string
   default     = "HSM"
 }
+
+variable "data_access_logs_enabled" {
+  description = "Enable Data Access logs of types DATA_READ, DATA_WRITE for all GCP services. Enabling Data Access logs might result in your organization being charged for the additional logs usage. See https://cloud.google.com/logging/docs/audit#data-access The ADMIN_READ logs are enabled by default."
```
Comment (erlanderlo) [inline modules/centralized-logging/variables.tf:79]: "... for all GCP services *in the projects under the provided `parent_folder`*. "

File: modules/centralized-logging/variables.tf [hunk 2]
```diff
@@ -40,6 +40,11 @@ variable "bucket_name" {
   type        = string
 }
 
+variable "parent_folder" {
+  description = "The folder id that will be logged."
+  type        = string
```
Comment (erlanderlo) [inline modules/centralized-logging/variables.tf:45]: what happens if I put an "" (empty) parent folder id as input?  (or all the projects are not in the same folder)

In other words, should we enforce any pre-condition check(s).  I think this is sorta coupled to the new boolean introduced.  This also seems to assume that all the projects will be in the same folder (i.e the governance, non-confidential, confidential, etc)

File: modules/centralized-logging/variables.tf [hunk 3]
```diff
@@ -40,6 +40,11 @@ variable "bucket_name" {
   type        = string
 }
 
+variable "parent_folder" {
+  description = "The folder id that will be logged."
```
Comment (erlanderlo) [inline modules/centralized-logging/variables.tf:44]: description is kinda confusing.  I think it's meant to be something like.

The folder id that will be configured so all its projects will add data logs for every resource.  (or something to that effect...I think?)

**Review by erlanderlo — COMMENTED** [review]
nice!  couple questions to consider.

**Review by erlanderlo — CHANGES_REQUESTED** [review]
Will be useful, thanks much!  Couple things to consider.

File: modules/centralized-logging/variables.tf (hunk 2 above)
Comment (LuizSDCit) [reply modules/centralized-logging/variables.tf:45]: The centralized-logging module is not a public module. We built it to be used only in the standalone examples. 

We do not consider those corner cases in the test.

Do you think that is an important check to do even in a private module?

File: modules/centralized-logging/variables.tf (hunk 1 above)
Comment (LuizSDCit) [reply modules/centralized-logging/variables.tf]: Good catch!

File: modules/centralized-logging/variables.tf (hunk 3 above)
Comment (LuizSDCit) [reply modules/centralized-logging/variables.tf]: Changed the description, I think that your suggestion make more sense.

File: modules/centralized-logging/variables.tf (hunk 2 above)
Comment (erlanderlo) [reply modules/centralized-logging/variables.tf:45]: +1 good reminder - not as important due to the coupling for now.

**Review by erlanderlo — APPROVED** [review]
LGTM

**Review by erlanderlo — APPROVED** [review]
thanks for making the changes. lgtm
